core/themes/basis/template.php

```diff
@@ -44,7 +44,22 @@ function basis_preprocess_layout(&$variables) {
 }
 
 /**
- * Implements template_preprocess_header().
+ * Prepares variables for node templates.
+ *
+ * @see node.tpl.php
+ */
+function basis_preprocess_node(&$variables) {
+  if ($variables['status'] == NODE_NOT_PUBLISHED) {
+    $name = node_type_get_name($variables['type']);
+    $variables['title_suffix'] = array(
+      '#type' => 'markup',
+      '#markup' => '<div class="unpublished-indicator">' . t('This !type is unpublished.', array('!type' => $name)) . '</div>',
```

Comment:
The whole string is run through` t()` already, do we need to do only the type-name, separately?